pom.xml

```diff
@@ -108,16 +112,8 @@
         <sonar.verbose>true</sonar.verbose>
 
         <jsr107.api.version>1.0.0</jsr107.api.version>
-        <CacheManagerImpl>com.hazelcast.cache.HazelcastCacheManager</CacheManagerImpl>
```

Comment:
Completely unused copy&paste stuff from Hazelcast (will hopefully be removed there soon as well).